### ISSUE TITLE
domain mismatch

### ISSUE THREAD

**Issue by jamesbeedy** [documize/community#120]
Getting this in the logs... can't seem to login after initial user creation.

```bash
Sep 23 20:13:41 ip-172-31-12-68 documize-community-linux-amd64[14363]: Processing migration file: db_00013.sql
Sep 23 20:13:41 ip-172-31-12-68 documize-community-linux-amd64[14363]: Processing migration file: db_00014.sql
Sep 23 20:13:41 ip-172-31-12-68 documize-community-linux-amd64[14363]: Processing migration file: db_00015.sql
Sep 23 20:13:42 ip-172-31-12-68 documize-community-linux-amd64[14363]: Database checks: completed
Sep 23 20:13:43 ip-172-31-12-68 documize-community-linux-amd64[14363]: logon attempt jamesbeedy@gmail.com @
Sep 23 20:13:43 ip-172-31-12-68 documize-community-linux-amd64[14363]: login jamesbeedy@gmail.com @
Sep 23 20:13:43 ip-172-31-12-68 documize-community-linux-amd64[14363]: domain mismatch 52 vs. 52 vs.
Sep 23 20:13:48 ip-172-31-12-68 documize-community-linux-amd64[14363]: logon attempt jamesbeedy@gmail.com @
Sep 23 20:13:48 ip-172-31-12-68 documize-community-linux-amd64[14363]: login jamesbeedy@gmail.com @
Sep 23 20:13:48 ip-172-31-12-68 documize-community-linux-amd64[14363]: domain mismatch 52 vs. 52 vs.
```

**Comment by HarveyKandola** [comment]
which version are you running?

**Comment by jamesbeedy** [comment]
@HarveyKandola 1.53.5

**Comment by HarveyKandola** [comment]
Fixed in 1.53.6 release!
